packages/shared/src/components/sidebar/DiscoverSection.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { ReactElement, useContext } from 'react';
+import FeaturesContext from '../../contexts/FeaturesContext';
+import DiscussIcon from '../icons/Discuss';
+import HotIcon from '../icons/Hot';
+import SearchIcon from '../icons/Search';
+import UpvoteIcon from '../icons/Upvote';
+import { ListIcon, SidebarMenuItem } from './common';
+import { Section, SectionCommonProps } from './Section';
+
+interface DiscoverSectionProps extends SectionCommonProps {
+  isItemsButton?: boolean;
+  enableSearch?: () => unknown;
+  onNavTabClick?: (page: string) => unknown;
+}
+
+export function DiscoverSection({
+  isItemsButton,
+  enableSearch,
+  onNavTabClick,
+  ...defaultRenderSectionProps
+}: DiscoverSectionProps): ReactElement {
+  const { popularFeedCopy } = useContext(FeaturesContext);
```

Comment:
One thing to consider is actually passing this down from the sidebar, we already initialise the FeaturesContext there so passing down would be neater?